### PR TITLE
[luci-interpreter] Enable While

### PR DESCRIPTION
**Pull request by seanshpark**
This will enable execution of While Op.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #7280, from draft #7358

File: compiler/luci-interpreter/src/kernels/While.test.cpp [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "core/RuntimeModule.h"
+#include "kernels/Add.h"
+#include "kernels/Less.h"
+#include "kernels/Minimum.h"
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/While.test.cpp:21]: Is `Minimum` used in this file?

File: compiler/luci-interpreter/src/kernels/While.h [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  const Tensor *input(int index) const { return _inputs[index]; }
+  Tensor *output(int index) const { return _outputs[index]; }
+
+  void configure() override;
+  void execute() const override;
+
+private:
+  RuntimeGraph *const _cond_graph;
+  RuntimeGraph *const _body_graph;
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/While.h:42]: Maybe for SE check?

```suggestion
  RuntimeGraph *const _cond_graph = nullptr;
  RuntimeGraph *const _body_graph = nullptr;
```

File: compiler/luci-interpreter/src/kernels/While.test.cpp [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "core/RuntimeModule.h"
+#include "kernels/Add.h"
```
Comment (seanshpark) [inline compiler/luci-interpreter/src/kernels/While.test.cpp:19]: This is also not needed -_-;

**Review by jinevening — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
@binarman , PTAL

**Review by binarman — APPROVED** [review]
LGTM
function comment is optional, it just improves readability a bit,

**Comment by seanshpark** [comment]
> function comment is optional, it just improves readability a bit,

Thanks for the approval, I'll work on extracting to a separate function after landing related codes :)

**Pull request merged**